python/python/glide/async_commands/cluster_commands.py

```diff
@@ -340,3 +340,30 @@ async def time(self, route: Optional[Route] = None) -> TClusterResponse[List[str
             TClusterResponse[List[str]],
             await self._execute_command(RequestType.Time, [], route),
         )
+
+    async def lastsave(self, route: Optional[Route] = None) -> TClusterResponse[int]:
```

Comment:
Maybe my idea violates the style already used there, but I think we need to split this into 2 commands:
```
async def lastsave(self) -> int:
async def lastsave(self, route: Route) -> TClusterResponse[int]:
```
Because without route it returns `int` (a single node response)

Reply:
We actually don't need to split it into 2 commands because of how TClusterResponse is defined:
`TClusterResponse = Union[T, Dict[str, T]]`
So the function signature allows for either an `int` (standalone) or a `Dict[str, int]` to be returned. This is what some of the other commands in this file are doing as well.

Reply:
Yes, I know. I dislike union, because it could be interpreted that `lastsave()` returns `Dict`. Do we need to add extra doc to be clear?

Reply:
The "Returns:" portion of the doc already clarifies that single node routes return an int but multi-node routes return a dict, and the other functions/docs follow this pattern, so personally I think its fine as is